src/components/AdminBoard.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="admin-board">
+    <reset-button
+      class="admin-button-top"
+      :activated="adminModeState[0]"
+      @reset="adminMode(0)"
+    />
+    <reset-button
+      class="admin-button-bottom"
+      :activated="adminModeState[1]"
+      @reset="adminMode(1)"
+    />
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+import ResetButton from './ResetButton.vue';
+
+export default {
+  components: {
+    ResetButton,
+  },
+  data() {
+    return {
+      adminModeState: [false, false],
+    };
+  },
+  methods: {
+    adminMode(playerId) {
+      Vue.set(this.adminModeState, playerId, !this.adminModeState[playerId]);
+
+      if (this.adminModeState.every((value) => value)) {
+        console.log('Entering admin mode');
+      }
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+  @use 'styles/constants';
+  @use 'styles/colors';
+
+  #admin-board {
+    display: grid;
+    justify-items: end;
+    grid:
+    "admin-button-top" constants.$reset-button-size
```

Comment:
You could just call these areas "top" and "bottom" since we are in the context of the admin board. The SideBoard specifies "reset-button" because there is other stuff between the reset buttons